kernel/kernel-defaults/src/test/java/io/delta/kernel/defaults/internal/parquet/TestParquetBatchReader.java

```diff
@@ -435,6 +435,40 @@ private static void verifyRowFromAllTypesFile(
         }
     }
 
+    private static void validateArrayOfArraysColumn(
```

Comment:
Can we also do Array(emptyArray) and Array(null)?

Reply:
Also maybe Map(int, Array) or something? Since the Map/Array implementation is different I think it's good to have a nested test for both Map/Array with one of their own